astronomer/providers/amazon/aws/example_dags/example_aws_nuke.py

```diff
@@ -125,6 +131,15 @@ def check_dag_status(**kwargs: Any) -> None:
         f"aws emr-containers list-virtual-clusters --state RUNNING --region {AWS_DEFAULT_REGION} | jq -r '.virtualClusters[].id' | xargs -I % aws emr-containers delete-virtual-cluster --id % --region {AWS_DEFAULT_REGION}; ",
     )
 
+    terminate_regression_clusters = BashOperator(
+        task_id="terminate_regression_clusters",
+        bash_command=f"set -e; "
+        f"aws configure set aws_access_key_id {REGRESSION_CLUSTER_AWS_ACCESS_KEY}; "
+        f"aws configure set aws_secret_access_key {REGRESSION_CLUSTER_AWS_SECRET_ACCESS_KEY}; "
+        f"aws configure set default.region {REGRESSION_CLUSTER_AWS_DEFAULT_REGION}; "
+        f"sh $AIRFLOW_HOME/dags/delete_eks_cluster_and_nodes.sh {REGRESSION_CLUSTER_AWS_DEFAULT_REGION}",
```

Comment:
do we need to pass `REGRESSION_CLUSTER_AWS_DEFAULT_REGION ` as a parameter can't we access it in the bash file directly with something like `${REGRESSION_CLUSTER_AWS_DEFAULT_REGION}`?